Add a changelog section, common to both variants (#28)
neo4j/docs-dataflow-connector: common-content-dataflow/modules/ROOT/partials/changelog.adoc

= Changelog

This page lists changes to the Dataflow Flex Template for BigQuery and/or Google Cloud to Neo4j.

You can check the deployment status of a release by running `gsutil ls gs://dataflow-templates/`.

Releases are gradually deployed, some regions may be updated before others.
To check the release status of https://cloud.google.com/compute/docs/regions-zones[a particular region], you can run `gsutil ls gs://dataflow-templates-$\{region}/` (e.g. `gsutil ls gs://dataflow-templates-europe-west2/`).

> 

Reviewer: Shall we also mention that it is not possible to run any version other than _latest_?

Author: Yes, let me edit this in

== Release 2025-06-10-01_RC00

* Add support to see underlying Neo4j driver's Bolt logs

== Release 2025-01-26-00_RC00

> * Add support for calendar-based version names of the Neo4j database

Reviewer: What does this mean? What are some examples of `version names` I would see?

Author: This refers to this change specifically:
https://github.com/GoogleCloudPlatform/DataflowTemplates/commit/e6a515b15a39d01da548c34c94e57d2db936a8e8

I.e. a small change to make sure the new version format of neo4j since version 2025 will still work.

Is this unnecessary information to highlight or should we re-formulate it a bit?

Reviewer: Ah, right, I see. I don't know if it's unnecessary information to highlight - if we do highlight it, I would phrase it as "Support for self-managed deployments of Neo4j 2025.x", because that's what's relevant to customers.

Author: Thank you so much for the input, I will add this!

== Release 2025-01-22-01_RC00

* Switch to Java 17

> * Updated Neo4j driver version from v4.4.18 to v5.27.0

Reviewer: Are we not yet running on 5.28.8 (i.e. the LTS)?

Author: It's a retroactive change-log, this date 2025-01-22, the LTS version 5.28.x did not exist yet.

We are still on v5.27.0, great observation -- maybe we should upgrade this to a later version, will hear with team

== Release 2025-01-14-00_RC00

* Re-added node target's existence constraint

== Release 2025-01-07-00_RC01

* Remove node target's existence constraint

> * Fix an issue with custom Cypher queries

Reviewer: I find these sort of entries quite useless from a user's perspective. Is it a very-hard-to-describe issue, or can we at least hint at what it affected?

Author: Did some git history research, read some code. Posting an update to what changed.

* Further minor bug-fixes

== Release 2024-12-10-00_RC00

> * Added support for BigQuery properties 'query_temp_project' and 'query_temp_dataset'

Reviewer: ```suggestion
* Added support for BigQuery properties `query_temp_project` and `query_temp_dataset`
```

== Release 2024-11-19-00_RC00

* Add YAML support when parsing specification

== Release 2024-11-12-00_RC00

* Empty strings are now treated as null values when importing from CSV files.

== Release 2024-09-18-00_RC00

> * Introduce new import specification (https://github.com/neo4j/import-spec)

Reviewer: ```suggestion
* Introduce new import specification (https://github.com/neo4j/import-spec), on which Dataflow templates base their import specifications.
```

= Changelog

This page lists changes to the Dataflow Flex Template for BigQuery and/or Google Cloud to Neo4j.

You can check the deployment status of a release by running `gsutil ls gs://dataflow-templates/`.

Releases are gradually deployed, some regions may be updated before others.
To check the release status of https://cloud.google.com/compute/docs/regions-zones[a particular region], you can run `gsutil ls gs://dataflow-templates-$\{region}/` (e.g. `gsutil ls gs://dataflow-templates-europe-west2/`).

> It is not possible to run any other version than the *latest* version of your region.

Reviewer: ```suggestion
It is not possible to run any version other than the *latest* one deployed in a given region.
```
